cookiecutter.json

```diff
@@ -14,6 +14,19 @@
   "license": ["MIT", "BSD-3", "GPL-3.0"],
   "funder": "JBFC: The Joe Bloggs Funding Council",
   "__prompts__": {
-    "deploy_docs_to_github_pages": "Enable automatically deploying HTML documentation to GitHub Pages website on pushes to main"
+    "author_given_names": "Given name(s) of package author",
+    "author_family_names": "Family name(s) of package author",
+    "author_email": "Email address for package author - will be publicly accessible",
+    "project_name": "Name for project - may contain spaces",
```

Comment:
```suggestion
    "project_name": "Name of project - may contain spaces",
```

Reply:
I guess this partly depends whether we think people will be running this having already thought of a project name (in which case I think _of_ would be correct) or will be thinking of a name on the spot (in which case _for_ seems natural to me). Probably realistically the former is more likely so will change!